index.html

```diff
@@ -205,21 +205,19 @@ <h3>Skills</h3>
               </ul>
             </div>
           </section>
-          
-		<section id="contact_form">
-			<div class="contact_detail">
-				<h2>Contact me</h2>
-				<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
-					I’d love to help with it!
-				</p>
-			</div>
-			<form action="https://formspree.io/f/xzbqlavz" method="POST" class="form">
-					<input class="input" type="text" name="text" maxlength="30" required placeholder="Enter your full name here">
-					<input class="input" type="email" name="email" required placeholder="Enter your email here">
-					<textarea class="input" name="message" placeholder="Enter your message here" maxlength="500" required></textarea>
-				  <button type="submit" class="button" >Get in touch</button>
-			</form>
-		</section>
+          <section class="form-section">
+            <h3>Contact me</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <form action="https://formspree.io/f/xzbqlavz" method="post">
+              <input type="text" maxlength="30" placeholder="Full Name" required>
+              <input type="email" name="email" id="" placeholder="Email address" required>
+              <textarea name="message" id="" maxlength="500" placeholder="Write your message here..." required></textarea>
```

Comment:
👍 Great job on adding the padding to the input elements! However, it seems that there is a small issue with the placeholder text. When the padding is applied, it introduces some extra space on the left side of the placeholder text. To resolve this, you can adjust the CSS styles to ensure that the placeholder text aligns properly within the input field, without any additional spacing.

| Your | Design|
| --- | --- |
| ![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/55840999/559337ed-6ec7-44e9-88c0-2f4ca668351c) | ![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/55840999/185ae80a-335d-4da8-a64c-d8e793c700fd) |

Reply:
@AbuTalha3 your current design still hasn't implemented this requested change by your previous reviewer, please make sure to add this changes before resubmitting  :+1: